benchmark/Snakefile

```diff
@@ -0,0 +1,537 @@
+            grep -v '#' $l | head -1
+        done | sed '1i#likelihood\tshape\tscale\ttheta' > {output[0]}
+        for l in {input.pop1_res}; do 
+            grep -v '#' $l | head -1
+        done | sed '1i#likelihood\tshape\tscale\ttheta' > {output[1]}
+        for l in {input.pop2_res}; do 
+            grep -v '#' $l | head -1
+        done | sed '1i#likelihood\tshape\tscale\ttheta' > {output[2]}
+        """
+
+#rule dadi_plot_results:
+#    input: rules.dadi_bestfit_dfe.output
+#    output:
+#        "results/dadi/pop0.two_epoch_1d.gamma.mut.prop.pdf"
+#    threads: 1
+#    shell:
+#        "dadi-cli Plot --dfe-popt {input} --pdf1d gamma --output {output} --mut-rate 2.35e-8 --seq-len 248956 --ratio 2.31"
+
+# ###############################################################################
+# polyDFE
+# ###############################################################################
+
+rule run_polydfe:
+    input: 
+        output_dir + "/polyDFE/{seeds}/{chrms}/pop{ids}/pop{ids}.polyDFE.fs"
+    output: 
+        output_dir + "/polyDFE/{seeds}/{chrms}/pop{ids}/pop{ids}.polyDFE.results"
+    threads: 1
+    resources: time_min=60, mem_mb=5000, cpus=1
+    shell:
+        "polyDFE-2.0-linux-64-bit -d {input} -m C -i config/polyDFE/polyDFE_init_models.txt 1 -e > {output[0]}"
+
+rule get_polydfe_bestfit:
+    input:
+        output_dir + "/polyDFE/{seeds}/{chrms}/pop{ids}/pop{ids}.polyDFE.results"
+    output:
+        output_dir + "/polyDFE/{seeds}/{chrms}/pop{ids}/pop{ids}.polyDFE.bestfit"
+    threads: 1
+    shell:
+        """
+        paste <(grep "Best joint likelihood" results/polyDFE/{wildcards.seeds}/{wildcards.chrms}/pop{wildcards.ids}/pop{wildcards.ids}.polyDFE.results | awk '{{print $6}}') \
+              <(grep eps_an results/polyDFE/{wildcards.seeds}/{wildcards.chrms}/pop{wildcards.ids}/pop{wildcards.ids}.polyDFE.results -A 3 | tail -1 | awk 'BEGIN{{OFS="\t"}}{{print $2,$3}}') \
+              <(grep eps_an results/polyDFE/{wildcards.seeds}/{wildcards.chrms}/pop{wildcards.ids}/pop{wildcards.ids}.polyDFE.results -A 1 | tail -1 | awk '{{print $3}}') > {output}
+        """
+
+rule combine_polydfe_bestfits:
+    input:
+        pop0_res = expand(output_dir + "/polyDFE/{seeds}/{chrms}/pop0/pop0.polyDFE.bestfit", seeds=seed_array, chrms=chrm_list),
+        pop1_res = expand(output_dir + "/polyDFE/{seeds}/{chrms}/pop1/pop1.polyDFE.bestfit", seeds=seed_array, chrms=chrm_list),
+        pop2_res = expand(output_dir + "/polyDFE/{seeds}/{chrms}/pop2/pop2.polyDFE.bestfit", seeds=seed_array, chrms=chrm_list)
+    output:
+        expand(output_dir + "/polyDFE/{chrms}/pop{ids}.polyDFE.bestfits", chrms=chrm_list, ids=[0, 1, 2])
+    threads: 1
+    shell:
+        """
+        cat {input.pop0_res} | sed '1i#likelihood\tS_d\tb\ttheta_bar' > {output[0]}
+        cat {input.pop1_res} | sed '1i#likelihood\tS_d\tb\ttheta_bar' > {output[1]}
+        cat {input.pop2_res} | sed '1i#likelihood\tS_d\tb\ttheta_bar' > {output[2]}
+        """
```

Comment:
Hi @chriscrsmith, here are the codes for DFE inference with polyDFE.
You can find the pre-complied binaries and manual in https://github.com/paula-tataru/polyDFE
The config file I used can be found in https://github.com/xin-huang/analysis2/blob/main/benchmark/config/polyDFE/polyDFE_init_models.txt